db/controller.js

```diff
@@ -33,7 +33,8 @@ exports.getArticlesById = (request, response, next) => {
 };
 
 exports.getArticles = (request, response, next) => {
-    selectArticles()
+    const { topic } = request.query;
+    selectArticles(topic)
```

Comment:
There should be a way of checking if the topic exists here, as we don't want to risk SQL injection.
Note: The existence of a topic must come from the db, not to be hard coded.